modules/electromagnetics/doc/content/source/auxkernels/AzimuthMagneticTimeDerivScalar.md

```diff
@@ -0,0 +1,28 @@
+# AzimuthMagneticTimeDerivScalar
+
+!syntax description /AuxKernels/AzimuthMagneticTimeDerivScalar
+
+## Overview
+
+!style halign=left
+The AzimuthMagneticTimeDerivScalar object calculates the time derivative of the azimuthal magnetic field for an axisymmetric cylindrical configuration. The term is define as:
+
+\begin{equation}
+  \frac{\partial \vec{B}_{\theta}}{\partial t} = -\left( \frac{\partial \vec{E}_{\rho}}{\partial z} - \frac{\partial \vec{E}_{z}}{\partial \rho} \right)
```

Comment:
is it evaluating the left side or the right side?
Is this just maxwell law?

Reply:
It is the right side (I can edit that for clarification). Yes, in particular it is Faraday's law of induction. This Auxkernel was used for experimental validation against B-Dot probe data, which measures the time derivative of the magnetic field.